apps/hestia/next.config.js

```diff
@@ -61,6 +61,7 @@ const nextConfig = {
     SHOW_SHUTDOWN_POPUP: process.env.SHOW_SHUTDOWN_POPUP,
     UNDER_MAINTENACE: process.env.UNDER_MAINTENACE,
     ENABLE_LMP: process.env.ENABLE_LMP,
+    ENABLE_BRIDGE: process.env.ENABLE_BRIDGE,
```

Comment:
invert this flag. instead of disabling by default, enable by default and give option to disable. ie make the env var IS_BRIDGE_ENABLED

Reply:
Done. Bridge page will be active always until we set `IS_BRIDGE_ENABLED` to `false` in env